zebrad/src/commands/start.rs

```diff
@@ -194,6 +195,8 @@ impl StartCmd {
                 .in_current_span(),
         );
 
+        let rpc_task_handle = tokio::spawn(RpcServer::new(config.rpc).in_current_span());
```

Comment:
Blocking Problem (3/3): Use the spawned task.

```suggestion
        let rpc_task_handle = RpcServer::spawn(config.rpc);
```

Reply:
https://github.com/ZcashFoundation/zebra/pull/3589/commits/8ed2047734f5915cc227f861d5e82991137d5ca2